CorsixTH/Lua/class.lua

```diff
@@ -65,8 +65,8 @@ local function define_class(name, super, adopts_self)
   local methods = {}
   local methods_mt = {}
 
-  local function new_class(methods, ...)
-    local constructor = methods[name]
+  local function new_class(methods2, ...)
```

Comment:
Not a fan of calling it methods2, as that implies it's different from methods. It's not, but call on a metatable passes the actual table as the first argument. ```t``` is a fairly conventional name for that first argument. ```m``` would be fine as well in our case.